src/modules/searchMoviesReducer.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import api from 'utils/Tmdb.js';

```

Comment:
I think, that better will be if this module will be for all the movies, not only for some functionality as searching.  In such way, you will have fewer modules in the application.

So naming `movies` will fit better.